lib/modules/dosomething/dosomething_reportback/dosomething_reportback.module

```diff
@@ -492,6 +492,17 @@ function _dosomething_reportback_get_permalink_copy_vars($user) {
  * Alters the signup form that is attached to permalink pages.
  */
 function dosomething_reportback_form_alter(&$form, &$form_state, $form_id) {
+
+  // Make sure a default run is added to the filter
+  if ($form['#id'] == 'views-exposed-form-reportback-files-page-1') {
+    if (empty($_GET['run_nid'])) {
+      $run = dosomething_helpers_get_current_campaign_run_for_user(arg(1));
+      if (isset($run)) {
```

Comment:
This PR is setting the value for this filter 

![screen shot 2016-02-19 at 3 26 59 pm](https://cloud.githubusercontent.com/assets/897368/13188096/5786cce2-d71d-11e5-8950-ad5e55f42f11.png)

However that should only happen when the filter is empty. If a value already exists in the filter, I don't want to replace it.